src/chi/javascript/core/util.js

```diff
@@ -533,6 +533,30 @@ export class Util {
     return null;
   }
 
+  static uuid4() {
+    let uuid = '', ii;
```

Comment:
Please, define each variable in its own line, and give a more meaningful name to _ii_ if possible. Thanks in advance